pex/bin/pex.py

```diff
@@ -290,6 +291,22 @@ def configure_clp_pex_options(parser):
         ),
     )
 
+    group.add_argument(
+        "--exclude",
+        dest="excluded",
+        default=[],
+        type=str,
+        action="append",
+        help=(
+            "Adds a requirement to exclude from the built PEX. Any distribution included in the "
```

Comment:
The wording I find could be a little misleading, as we're only adding to the list of things to exclude, and it is the excluding bit that is interesting. (I think the note about this option being able to be used multiple times at the end of the help text to be sufficient regarding the additiveness of the option.)

How about something like `"Names a requirement to exclude ..."` instead?

Reply:
Ok. I guess the other "adds" you find misleading is thinking this adds the requirement to the PEX?

I think `s/Names/Specifies/` sits better with me since a requirement in a name + a version sepcifier and although most people proably will just use the name part, they really can specify a full requirement, including marker - for example to exclude only for mac in a multiplatform PEX for example. Does that work for you?

Reply:
Specifies works just as well, yes 👍🏽 

Yes, it's a bit of a double negative with "add to exclude". I did understand it, but can also see it being an easy mistake to make. (I may have paused and read the sentence a second time to make sure I parsed it correctly..)

Reply:
Fixed.